cylc/flow/scheduler_cli.py

```diff
@@ -42,67 +42,48 @@
 from cylc.flow import suite_files
 from cylc.flow.terminal import cli_function
 
-RUN_DOC = r"""cylc [control] run|start [OPTIONS] [ARGS]
+PLAY_DOC = r"""cylc [control] play [OPTIONS] [ARGS]
 
-Start a new suite run.
-
-Any dependencies prior to the start point will be ignored.
-
-WARNING: this will wipe out previous suite state. To restart from a previous
-state, see 'cylc restart --help'.
+Start running a workflow, or restart a stopped workflow from its previous
+state/cycle point, or resume a paused workflow by releasing all tasks.
 
 The scheduler will run as a daemon unless you specify --no-detach.
 
-If the suite is not already installed (by "cylc install" or a previous run)
+If the workflow is not already installed (by "cylc install" or a previous run)
 it will be installed on the fly before start up.
 
 Examples:
-    # Run the suite installed with name REG.
-    $ cylc run REG
-
-    # Install $PWD/flow.cylc as $(basename $PWD) and run it.
-    # Note REG must be given explicitly if START_POINT is on the command line.
-    $ cylc run
-
-A "cold start" (the default) starts from the suite initial cycle point
-(specified in flow.cylc or on the command line). Any dependence on tasks
-prior to the suite initial cycle point is ignored.
-
-A "warm start" (-w/--warm) starts from a given cycle point later than the suite
-initial cycle point (specified in flow.cylc). Any dependence on tasks prior
-to the given warm start cycle point is ignored. The suite initial cycle point
-is preserved."""
+    # Start/restart the workflow with name REG.
+    $ cylc play REG
 
-RESTART_DOC = r"""cylc [control] restart [OPTIONS] ARGS
+    # Install $PWD/flow.cylc as $(basename $PWD) and start it.
+    $ cylc play
```

Comment:
So I should make the `[REG]` arg non-optional?

Reply:
Yes please, it no longer makes sense since the change from registration to installation.

This feature will now only work if you are in the run directory for a suite which has either been written there or `cylc installed` and will fail in unhelpful ways otherwise.